### DISCUSSION TITLE
Chrome 124 已经实验性支持“通用安装”，可以安装任意网页

### DISCUSSION THREAD

**Discussion by nmnmcc** [tw93/Pake#704] [General]
https://developer.chrome.com/blog/new-in-chrome-124#more

> And universal install allows users to install any page, even those not meeting the [current PWA criteria](https://web.dev/articles/install-criteria).

https://www.androidpolice.com/any-website-pwa-chrome-124/

> In Chrome Canary (the daily build version of Google Chrome and typically a couple of versions ahead of the stable build), websites can now be installed on desktops. As part of the latest daily build, Google has added an "Install page as app…" option to the "Save and share" submenu on the desktop version (via [@Leopeva64 on X](https://twitter.com/Leopeva64/status/1765698342191780066/)). This makes clicking the app — which is just the website made to look and feel like a native app — always open in its own window. Sites that already have their own PWAs, like YouTube or Reddit, have been prompting users to install them for a while now and will have their "Install page as app…" function actually showing the name of the site. For example, YouTube's entry will show as "Install YouTube."

> To enable this feature in Chrome Canary now and test it, simply [install Canary](https://www.google.com/chrome/canary/) and enable both of the flags below:
>
> chrome://flags/#web-app-universal-install
>
> chrome://flags/#shortcuts-not-apps

此项目存在意义将会减少

**Comment by wumingluren** [comment]
还有打包本地版的需求

**Comment by tw93** [comment]
其实不会的，其实好比PWA出现了，还是会有很多这种场景
